frontend/src/components/GoalsTable/ObjectiveRow.js

```diff
@@ -106,24 +106,24 @@ function ObjectiveRow({
             Collapse objective(s)
           </button>
           {title}
-        </td>
-        <td>
+        </li>
+        <li>
           {' '}
           <Link
             to={linkToAr}
           >
             {arNumber}
           </Link>
-        </td>
-        <td>{endDate}</td>
-        <td>
+        </li>
+        <li>{endDate}</li>
```

Comment:
I haven't fired this up in a screenreader yet but before I do, each of these list items will need something like
```jsx
<li><span className="sr-only">End date:</span>{endDate}</li>
```
same for all the other objective fields.